sys/kern/tmpfs.c

```diff
@@ -743,7 +772,7 @@ static int tmpfs_mount(mount_t *mp) {
   /* Allocate the root node. */
   vattr_t va;
   vattr_null(&va);
-  va.va_mode = S_IFDIR | (ACCESSPERMS & ~CMASK);
+  va.va_mode = S_IFDIR | (ACCESSPERMS & ~CMASK) | S_ISVTX;
```

Comment:
Why sticky bit here?

Reply:
Removed. I added it because in every UNIX like system tmp dir is marked with sticky bit. However, there is no meaning for sticky bit at all in mimiker, so I will add it in future changes.